### PR TITLE
Improved test coverage for adding messaging bridge

### PR DESCRIPTION
**Pull request by honza-kasik**
Also improved ElytronIntegrationChecker API

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationChecker.java [hunk 1]
```diff
@@ -285,4 +435,45 @@ public ModelNode build() {
...
+
+        /**
+         * Config fragment containing credential-reference input fields
+         */
+        public Builder configFragment(ConfigFragment configFragment) {
+            this.configFragment = configFragment;
+            return this;
+        }
+
+        public ElytronIntegrationChecker build() {
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationChecker.java]: Please add validation that mandatory fields are set.

File: common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationChecker.java [hunk 2]
```diff
@@ -38,22 +38,166 @@
...
         this.operations = new Operations(client);
     }
 
+    public ElytronIntegrationChecker(Builder builder) {
+        this.client = builder.client;
+        this.operations = new Operations(client);
+        this.address = builder.address;
+        this.configFragment = builder.configFragment;
+        if (builder.credentialReferenceAttributeName == null) {
+            this.credentialReferenceAttributeName = CREDENTIAL_REFERENCE;
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationChecker.java]: Move this logic to the `Builder#build` method please.

**Review by zebrik — COMMENTED** [review]
Deprecated methods should be removed ASAP in one of upcoming PRs.

**Comment by zebrik** [comment]
LGTM. Please squash.

**Comment by honza-kasik** [comment]
Squashed, thanks!

**Pull request merged**